app/views/admin/merchants/show.html.erb

```diff
@@ -0,0 +1,2 @@
+<p><strong>Name:</strong> <%= @merchant.name %></p>
```

Comment:
what is a strong div>

Reply:
`<strong>` bolds HTML text. `<b>` works, too.